### PR TITLE
*: use terror for some error

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 
reduce some TODO: add terror

### What is changed and how it works?
use terror instead of error

### Check List 

Tests 

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: pkg/ha/stage.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ package ha
 import (
 	"context"
 	"encoding/json"
-	"fmt"
+	"github.com/pingcap/dm/pkg/terror"
```
Comment (csuzhangxc) [inline pkg/ha/stage.go:19]: re-order it.

File: pkg/ha/subtask.go [hunk 2]
```diff
@@ -15,7 +15,7 @@ package ha
 
 import (
 	"context"
-	"fmt"
+	"github.com/pingcap/dm/pkg/terror"
```
Comment (csuzhangxc) [inline pkg/ha/subtask.go:18]: ditto.

File: pkg/ha/task.go [hunk 3]
```diff
@@ -15,7 +15,7 @@ package ha
 
 import (
 	"context"
-	"fmt"
+	"github.com/pingcap/dm/pkg/terror"
```
Comment (csuzhangxc) [inline pkg/ha/task.go:18]: ditto.

File: pkg/ha/stage.go (hunk 1 above)
Comment (lance6716) [reply pkg/ha/stage.go]: https://github.com/pingcap/dm/pull/848/commits/c6e4bf9a5c25b7f9cdb9cd4c73ddea768e72f6f2

do you have any tools recommended to handle grouping-import 😂

Comment (csuzhangxc) [reply pkg/ha/stage.go]: not found yet, 😢

**Pull request merged**